eng/BootStrapMSBuild.targets

```diff
@@ -79,7 +79,8 @@
       <ShimTargets Include="Workflow.Targets" />
       <ShimTargets Include="Workflow.VisualBasic.Targets" />
 
-      <InstalledMicrosoftExtensions Include="$(MSBuildExtensionsPath)\Microsoft\**\*.*" />
+      <InstalledMicrosoftExtensions Include="$(MSBuildExtensionsPath)\Microsoft\**\*.*"
+                                    Exclude="$(MSBuildExtensionsPath)\Microsoft\VisualStudio\NodeJs\**" />
```

Comment:
Do you not have to add \*.*?

Reply:
Alternatively, I think you can replace these with the original:
`...Include=...\*.targets`
and
`...Include=...\*.props`

Reply:
No, you don't have to add `*.*`; that's redundant with `**`.

> Alternatively, I think you can replace these with the original: `...Include=...\*.targets` and `...Include=...\*.props`

Why do you think that? That would copy only `.targets` and `.props` files, but other files are part of these directories.